students/hyunyoung/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name = models.CharField(max_length=45)
+    email = models.CharField(max_length=100)
+    password = models.CharField(max_length=100)
+    phone_name = models.CharField(max_length=45)
```

Comment:
보다 명확한 변수 네이밍이 필요할 것 같네요~
```suggestion
    phone_number = models.CharField(max_length=45)
```